.github/workflows/podman-desktop-e2e-stress-ui-linux.yaml

```diff
@@ -0,0 +1,177 @@
+name: Podman Desktop E2E UI Stress Test (Linux)
+run-name: Podman Desktop E2E UI Stress Test (Linux)
+
+on:
+  schedule:
+    - cron: '0 5 * * 1' #run every monday at 5AM
+  workflow_dispatch:
+    inputs:
+      pd_repo_options:
```

Comment:
Except this and the npm_target input you do not use any of them at all. Clean that up please.